apps/studio/src/pages/home/uploadSong/BasicSongDetails.tsx

```diff
@@ -282,6 +282,7 @@ const BasicSongDetails: FunctionComponent<BasicDonDetailsProps> = ({
               emptyMessage="Drag and drop or browse your image"
               maxFileSizeMB={ 10 }
               minDimensions={ { height: 1400, width: 1400 } }
+              minFileSizeMB={ 0.1 }
```

Comment:
@wlara If we store a cloudinary URL in our database and the user has uploaded a 100kb png that's 1600x1600, won't cloudinary serve us a resized 1400x1400 that could be less than 100kb?